flare/output.py

```diff
@@ -41,19 +41,23 @@ class Output:
 
     def __init__(self, basename: str = 'otf_run',
                  verbose: str = 'INFO',
+                 print_xyz: bool = False,
                  always_flush: bool = False):
         """
         Construction. Open files.
         """
         self.basename = f"{basename}"
+        self.print_xyz = print_xyz
```

Comment:
If you agree with the `print_as_xyz` change in `gp_from_aimd.py` then we can refactor it here. I think that it makes it slightly more clear that it is changing the format of the printing to call it print *as* xyz, instead of printing xyz format in addition to what is ordinarily printed.

Reply:
cool. changed in e235d15a17aeda3065bb27a9d290dc06209a9ff0